app/policies/furniture_placement_policy.rb

```diff
@@ -1,14 +1,26 @@
 class FurniturePlacementPolicy < ApplicationPolicy
   alias furniture_placement object
+  delegate :space, to: :furniture_placement
+
+  class Scope
+    attr_accessor :person, :scope
+    def initialize(person, scope)
+      @person = person
+      @scope = scope
+    end
+
+    def resolve
+      # TODO: is this even the right scope?
+      @scope.joins(:room).where(room: { space: @person.spaces })
```

Comment:
We probably want to use the `RoomPolicy::Scope` to do this: `where(room: RoomPolicy::Scope.new(person, Room.all).resolve)`

Reply:
Done.